lib/Target/SubstraitPB/Export.cpp

```diff
@@ -223,6 +224,49 @@ FailureOr<std::unique_ptr<Rel>> SubstraitExporter::exportOperation(CrossOp op) {
   return rel;
 }
 
+FailureOr<std::unique_ptr<Rel>>
+SubstraitExporter::exportOperation(UnionDistinctOp op) {
```

Comment:
I am trying to keep the order of the functions, the forward declarations/macros, and template parameter lists ordered. Could you move this and the other additions to the right place?

Reply:
I will apply this feedback in the export function PR(https://github.com/substrait-io/substrait-mlir-contrib/pull/23).